CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## Unreleased
+
+## Bootes - 16-08-2019
+### Added
+- New plugin architecture for the following Asperathos components:
+   - Monitor
+   - Manager
+   - Visualizer
+   - Controller
+- Persistent plugins for the following Asperathos components:
+   - Monitor
+   - Manager
+   - Visualizer
+   - Controller
+- PID scalling strategy at Asperathos Controller 
```

Comment:
Rephrase to "PID scaling strategy on the Controller component". Same as above, we know everything here is Asperathos related :)